@xen-orchestra/web/src/App.vue

```diff
@@ -23,6 +24,18 @@ const uiStore = useUiStore()
 const { locale } = useI18n()
 const { get } = useCookies()
 
+// TODO: Remove when we considere XO6 in more advanced state
+const isFirstConnection = useLocalStorage('first-connection', true)
+if (isFirstConnection.value) {
+  const open = useModal({
+    component: import('@/components/modals/FirstConnection.vue'),
+    onConfirm: () => {
+      isFirstConnection.value = false
+    },
+  })
+
+  open()
+}
```

Comment:
```suggestion
// TODO: Remove when we consider XO6 in more advanced state
const isFirstConnection = useLocalStorage('first-connection', true)

const openModal = useModal({
  component: import('@/components/modals/FirstConnection.vue'),
  onConfirm: () => {
    isFirstConnection.value = false
  },
})

whenever(() => isFirstConnection.value, openModal, {
  immediate: true,
})
```

To continue using vueUse, I think we could do this. What do you think?